scripts/deploy/contracts/InvestorVesting.ts

```diff
@@ -6,39 +6,151 @@ import hre, { ethers } from "hardhat";
+    initialAmount: ethers.utils.parseEther("0"),
+    initialClaimed: true,
+    claimStartTime: startDate + 0
+  },
+  {
+    beneficiary: "0x6b441cC275D3088A62dD12609acEa6f446C645b6",
+    vestingAmount: ethers.utils.parseEther("2000000000"),
+    duration: 157680000,
+    claimedAmount: 0,
+    lastClaimedTime: 0,
+    initialAmount: ethers.utils.parseEther("0"),
+    initialClaimed: true,
+    claimStartTime: startDate + 0
+  },
+  {
+    beneficiary: "0xD8C128747183AF6EbdA860A2ea9B77b3933c25b4",
+    vestingAmount: ethers.utils.parseEther("500000000"),
+    duration: 157680000,
+    claimedAmount: 0,
+    lastClaimedTime: 0,
+    initialAmount: ethers.utils.parseEther("0"),
+    initialClaimed: true,
+    claimStartTime: startDate + 31536000
+  },
+  {
+    beneficiary: "0xfa0B65413E3E81FAF7321f85b2AfdD0EffF13Ef5",
+    vestingAmount: ethers.utils.parseEther("350000000"),
+    duration: 63072000,
     claimedAmount: 0,
     lastClaimedTime: 0,
-    initialAmount: ethers.BigNumber.from("0"),
+    initialAmount: ethers.utils.parseEther("0"),
     initialClaimed: true,
-    claimStartTime: startDate + 1800
+    claimStartTime: startDate + 0
+  },
+  {
+    beneficiary: "0xfB98a42A0eb71470A9801e10E64B69f5A68e0024",
+    vestingAmount: ethers.utils.parseEther("50000000"),
+    duration: 31536000,
+    claimedAmount: 0,
+    lastClaimedTime: 0,
+    initialAmount: ethers.utils.parseEther("5000000"),
+    initialClaimed: false,
+    claimStartTime: startDate + 0
+  },
+  {
+    beneficiary: "0x73C2006262Ab12962120BEC17d1815A0EB845e76",
+    vestingAmount: ethers.utils.parseEther("250000000"),
+    duration: 31536000,
+    claimedAmount: 0,
+    lastClaimedTime: 0,
+    initialAmount: ethers.utils.parseEther("25000000"),
+    initialClaimed: false,
+    claimStartTime: startDate + 0
+  },
+  {
+    beneficiary: "0xAec7D36147963706Dbc79e371f9Bcb822F10D60D",
+    vestingAmount: ethers.utils.parseEther("50000000"),
+    duration: 31536000,
+    claimedAmount: 0,
+    lastClaimedTime: 0,
+    initialAmount: ethers.utils.parseEther("5000000"),
+    initialClaimed: false,
+    claimStartTime: startDate + 0
   },
   {
-    beneficiary: "0xd7016BBF436fea30f2a50eF0D56397a983be4129",
-    vestingAmount: ethers.BigNumber.from("5000000"),
-    duration: 7200,
+    beneficiary: "0xd3a911E2586ABC81f0B1e6Ec5AfC61899dc816B9",
+    vestingAmount: ethers.utils.parseEther("250000000"),
+    duration: 31536000,
     claimedAmount: 0,
     lastClaimedTime: 0,
-    initialAmount: ethers.BigNumber.from("500000"),
+    initialAmount: ethers.utils.parseEther("25000000"),
     initialClaimed: false,
-    claimStartTime: startDate
+    claimStartTime: startDate + 0
   },
   {
-    beneficiary: "0xEBE3efcfD01e27c1Ea8E48f5d55550C6325237af",
-    vestingAmount: ethers.BigNumber.from("30000000"),
-    duration: 18000,
+    beneficiary: "0xF2aE68ee7F461Ed24f96D089b6b48579EDbd7E4a",
+    vestingAmount: ethers.utils.parseEther("60000000"),
+    duration: 31536000,
```

Comment:
Should include cliff 
```suggestion
    duration: 63072000,
```